go.mod

```diff
@@ -51,7 +51,6 @@ require (
 require (
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/cespare/xxhash/v2 v2.2.0 // indirect
-	github.com/coreos/license-bill-of-materials v0.0.0-20190913234955-13baff47494e // indirect
```

Comment:
These come from a go mod tidy. I believe we don't have that check in release-3.4. Thanks for running it, Mustafa.